### ISSUE TITLE
"Using" sound of permanent target/newTarget object only plays if the transition has no newActor

### ISSUE THREAD

**Issue by UncleGus** [jasonrohrer/OneLife#28]
Using sound should play if the newActor has no creation sound, or newActor's creation sound is set to initial only.

In my example, I have an object called "Hot Copper Ingot On Copper Anvil#hammer,working" which is permanent and has a using sound. I have a transition of "@ Hammer" on "Hot Copper Ingot On Copper Anvil#hammer,working" = "@ Hammer" and "Hot Copper Ingot On Copper Anvil#hammer,working", which consumes one use of the "Hot Copper Ingot On Copper Anvil#hammer,working" object, but no sound plays. When I set the newActor of the transition to empty, then the using sound of the "Hot Copper Ingot On Copper Anvil#hammer,working" plays.

I think this might be affecting the goose ponds in the base game, as they have a "splash" using sound, which is not played when using a bucket on it, for example.

I also realise that asking you to fix this for my mod is pretty selfish, so don't treat this as high priority :)

**Comment by UncleGus** [comment]
I found the code responsible in line `8769` of `LivingLifePage.cpp`, and there is a comment, but I don't understand the intention:
```
                                        // don't play using sound
                                        // if they are both use dummies
                                        if( !bothSameUseParent( old, newID )
                                            &&
                                            obj->usingSound.numSubSounds 
                                            > 0 ) {    
                                            
                                            playSound( 
                                                obj->usingSound,
                                                getVectorFromCamera( x, y ) );
                                            }
                                        }
```
If we are using an object, and the interaction causes the number of uses of that object to change, why would we not want to play the "using" sound? I removed this check from my client, and now the goosepond plays the slosh sound when water is taken from or added to it.

Is it because this causes two sounds to be played from the one interaction? Is that an undesirable thing?

**Comment by jasonrohrer** [comment]
This was broken in v66 (it works in v65).  The change had something to do with murder weapon sounds.  I'm looking into it.

**Comment by jasonrohrer** [comment]
Fixed in commit 986099a118ade0e907e87d095ed5c35c7e98a749
